Makefile

```diff
@@ -170,6 +170,8 @@ check:
 	@! ineffassign . | grep -vF gossip/gossip.pb.go
 	@echo "errcheck"
 	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
+	@echo "returncheck"
+	@returncheck $(PKG)
```

Comment:
Some of the checks use `PKG` while others check all of the source. Doesn't that seem strange and inconsistent? Probably not worth fixing, though.